src/Sentry.Unity.Editor/Android/AndroidManifestConfiguration.cs

```diff
@@ -227,35 +230,64 @@ private string SaveAs(string path)
 
     internal class AndroidManifest : AndroidXmlDocument
     {
+        private const string _prefix = "io.sentry";
         private readonly XmlElement _applicationElement;
+        private readonly IDiagnosticLogger? _logger;
 
-        public AndroidManifest(string path) : base(path) =>
+        public AndroidManifest(string path, IDiagnosticLogger? logger) : base(path)
+        {
             _applicationElement = (XmlElement)SelectSingleNode("/manifest/application");
+            _logger = logger;
+        }
+
+        internal void RemovePreviousConfigurations()
```

Comment:
Do we need to remove the configuration? Could we just override the options instead?

Reply:
From my comment in the issue:

> It should first delete all "io.sentry" elements before starting to add new ones - this ensures no old configs remain from the previous run while being disabled in the current one. This would be preferable because some settings in the mentioned class are only set conditionally, so if they were previously set and the condition doesn't match anymore, they'd remain set on subsequent builds even though they wouldn't be if it was a clean build.